src/modules/home/_main.scss

```diff
@@ -103,6 +103,7 @@ html[lang^="en-"].theme-dark .container-2cd8Mz{
         &::before{
             @include FontIconFluent;
             color: $TextFillColorSecondary;
+            content: "\e711"; // Cancel
```

Comment:
This is kind of 'brute forcing' the icon change and would cause it to apply by default to any icons which aren't defined in the icons file (a bit messy).
Remove this line and add the below to `src\modules\icons\home\_main.scss`:
```
[aria-label="Cancel"]::before{
        content: "\E711"; // Cancel
    }
```

Side note: `#{map-get($translate, FRIEND_REQUEST_IGNORE)}` can be ignored for now.